helm/idol-nifi/README.md.gotmpl

```diff
@@ -26,6 +26,58 @@ Provides a scaleable IDOL NiFi cluster instance (NiFi, NiFi Registry and ZooKeep
 
 {{ template "chart.requirementsSection" . }}
 
+## Deploying a flow into NiFi 
+
+There are three methods for deploying a flow in a Nifi cluster:
+
+* Manually creating a flow once the NiFi cluster has been deployed by specifying no flows to be deployed in the NiFi cluster.
+
+```
+nifi:
+  flows: []
+```
+
+* Providing a JSON flow file (or multiple flow files) to be deployed. Specify the location of the JSON flow file to be deployed and the name of the NiFi Registry bucket the flow should be imported into prior to creating a version controlled process group within NiFi for the flow. If a flow of the same name already exists within the NiFi Registry bucket, and the provided flow is different to the latest version of the existing flow, a new version of the flow is created in NiFi Registry.
+
+```
+nifi:
+  flows:
+  - file: "/flows/flow1.json"
+    bucket: "my-bucket"
```

Comment:
I'm tempted to say we just shouldn't document the 'file' based import and just leave it there internally for 'flowfile' compatibility, whilst pushing people toward name/bucket/version registry-based import instead.